### PR TITLE
Update LocalProcessor to support FileSystemSink

### PR DESCRIPTION
**Pull request by Sxnan**
## What is the purpose of the change

Update LocalProcessor to support FileSystemSink.

## Brief change log
- Update LocalProcessor to support FileSystemSink.

## Does this pull request potentially affect one of the following parts:

  - Dependencies (does it add or upgrade a dependency): no
  - The public API: no

## Documentation

  - Does this pull request introduce a new feature? no
  - If yes, how is the feature documented? not applicable

### CODE REVIEW

**Comment by Sxnan** [comment]
@yunfengzhou-hub Could you review this PR?

File: python/feathub/processors/spark/tests/test_spark_processor.py
Comment (yunfengzhou-hub) [inline python/feathub/processors/spark/tests/test_spark_processor.py:1]: There is a `test_file_source` test case in `test_local_processor.py`. Could you please remove and replace it with the unified test cases in `test_file_system_source_sink.py` as well, like what this PR does for `test_spark_processor.py`?

File: python/feathub/processors/flink/table_builder/tests/test_flink_processor.py [hunk 1]
```diff
@@ -445,27 +444,6 @@ def get_client(self, extra_config: Optional[Dict] = None) -> FeathubClient:
...
-
-        files = glob.glob(f"{sink_path}/*")
-        df = pd.DataFrame()
-        for f in files:
-            csv = pd.read_csv(f, names=["name", "cost", "distance", "time"])
-            df = df.append(csv)
-        df = df.sort_values(by=["time"]).reset_index(drop=True)
-        self.assertTrue(self.input_data.equals(df))
-
     def test_unsupported_file_format(self):
```
Comment (yunfengzhou-hub) [inline python/feathub/processors/flink/table_builder/tests/test_flink_processor.py:447]: Should we also verify this test case against LocalProcessor and SparkProcessor, and do you think we should add support for remote FS for LocalProcessor in this PR?

**Review by yunfengzhou-hub — COMMENTED** [review]
Thanks for the PR. Left some comments as below.

File: python/feathub/processors/flink/table_builder/tests/test_flink_processor.py (hunk 1 above)
Comment (Sxnan) [reply python/feathub/processors/flink/table_builder/tests/test_flink_processor.py:447]: I think the behavior in this test is specific to the FlinkProcessor. 

Given that LocalProcessor is only used for local development and testing, I am not sure if we should support remote FS for LocalProcessor.  Anyway, I think supporting remote FS for LocalProcessor is out of the scope of this PR.

**Comment by yunfengzhou-hub** [comment]
Thanks for the update. LGTM. @lindong28 could you please take a look at this PR?

**Review by lindong28 — COMMENTED** [review]
Thanks for the PR. LGTM.

**Pull request merged**